deploy/docker-compose.yml

```diff
@@ -64,6 +66,16 @@ services:
       - RUST_LOG=debug
 
 
+  fuzon:
+    build: ./fuzon
+    volumes:
+      - ./fuzon/config.json.template:/fuzon/config.json.template:ro
```

Comment:
**praise**: never saw this notation for mounts where there's three colons and TIL that you can set permissions